HymnsTests/Display/DisplayHymnBottomBarViewModelSpec.swift

```diff
@@ -140,15 +140,12 @@ class DisplayHymnBottomBarViewModelSpec: QuickSpec {
                 }
                 let mp3Url = URL(string: "http://www.hymnal.net/en/hymn/h/1151/f=mp3")!
                 it("should have \(DisplayHymnBottomBarViewModel.overflowThreshold) buttons in the buttons list and nothing in the overflow") {
-                    expect(target.buttons).to(haveCount(7))
+                    expect(target.buttons).to(haveCount(5))
```

Comment:
Can you please update this test so that it has 6 elements? Then assert that the overflow is nil. It's meant to test the case where you hit the max number of options without going to overflow.

Reply:
Lol, I can't seem to get this right. So annoying, no matter what I add or subtract I am getting 5 elements. I don't see how before I changed it we had 7 and now we have 5.

Reply:
ok, I fixed it. It looks like the minimum number of buttons we have now is 6, so we can just remove this test and use the below test instead.